src/engine/fileOps.cpp

```diff
@@ -1591,17 +1593,20 @@ bool DivEngine::loadFur(unsigned char* file, size_t len) {
     }
 
     if (active) quitDispatch();
+    printf("loadFur active %d\n", active);
```

Comment:
logV

Reply:
Oop, leftover debug statement, shouldn’t be there at all!

Reply:
Removed.